google/resource_compute_instance_group_migrate_test.go

```diff
@@ -108,6 +104,10 @@ func TestComputeInstanceGroupMigrateState_empty(t *testing.T) {
 		is = &terraform.InstanceState{}
 		is, err = resourceComputeInstanceGroupMigrateState(tc.StateVersion, is, meta)
 
+		// should handle non-nil but empty
```

Comment:
I think these few lines can be removed (they're the same as what's directly above them)